Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get install -y \
 	python-pip
 	
 RUN pip install fabric --upgrade \
-	Requests --upgrade
+	pip install requests --upgrade \
```

Comment:
That does not count, been explicit about what to install rather than saving lines allow newbies to understand what is happening under the hood